sktime/benchmarking/evaluation.py

```diff
@@ -7,14 +8,18 @@
 import numpy as np
 import pandas as pd
 from scipy import stats
-from scipy.stats import ranksums
-from scipy.stats import ttest_ind
+from scipy.stats import ranksums, ttest_ind
+
 from sktime.benchmarking.base import BaseResults
 from sktime.exceptions import NotEvaluatedError
+from sktime.utils.check_imports import _check_imports
 
 plt.style.use("seaborn-ticks")
 
 
+_check_imports("scikit_posthocs")
```

Comment:
Looks like we're also importing matplolib in line 7 above and use it in line 17, so we need to move the import check a bit further up